libs/SalesforceSDK/src/com/salesforce/androidsdk/security/SalesforceKeyGenerator.java

```diff
@@ -73,8 +73,20 @@
      * @return Unique ID.
      */
     public static synchronized String getUniqueId(String name) {
```

Comment:
Remove `synchronized` keyword from this method and move it to the method below.

Reply:
Also, could you add a comment in the `Javadoc` for this method that the default key length used is `256`?

Reply:
Any specific reason to move it to the inside method? Just curious